client/common/directives/ModelFieldPointsOfInterest/ModelFieldPointsOfInterest.js

```diff
@@ -113,15 +109,16 @@ angular.module('dashboard.directives.ModelFieldPointsOfInterest', [
 				scope.boundaries = [];                 // Stored google boundary circles
 				scope.searchResults = [];              // All data recieved from query
 				scope.displayedMarkers = [];           // Markers that match query
-				scope.displayedSearchResults = [];     // Data for location list
+				scope.displayOptions = scope.property.display.options;
```

Comment:
Where is this used?

Reply:
Its not used anymore, ill remove it. Thanks